### PR TITLE
Hook up to Dependabot

### PR DESCRIPTION
**Pull request by jlebon**
See https://github.com/coreos/fedora-coreos-tracker/issues/960.

### CODE REVIEW

**Comment by jlebon** [comment]
Haven't tested this, just wrote it up based on docs and other Dependabot files.

**Comment by jlebon** [comment]
Meh... let's just try it and iterate on it?

**Review by cgwalters — APPROVED** [review]

**Comment by jlebon** [comment]
OK cool, looks like this works:

```
updater | INFO <job_203769950> Starting update job for coreos/fedora-coreos-stream-generator
updater | INFO <job_203769950> Checking if github.com/coreos/stream-metadata-go 0.0.0-20210225230131-70edb9eb47b3 needs updating
updater | INFO <job_203769950> Latest version is 0.0.0-20210225230131-70edb9eb47b3
updater | INFO <job_203769950> No update needed for github.com/coreos/stream-metadata-go 0.0.0-20210225230131-70edb9eb47b3
updater | INFO <job_203769950> Finished job processing
```

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+- package-ecosystem: gomod
+  directory: "/"
+  schedule:
+    interval: weekly
```
Comment (bgilbert) [inline .github/dependabot.yml:6]: We usually use `daily`, which seems like it'd make more sense for the problem we're trying to solve.

Comment (jlebon) [reply .github/dependabot.yml:6]: Thanks, filed https://github.com/coreos/fedora-coreos-stream-generator/pull/28. At first I hadn't realized that it was the only module so didn't want Dependabot to be too spammy.

**Pull request merged**